modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/online/AbstractWfsDataStoreOnlineTest.java

```diff
@@ -327,29 +332,6 @@ public void testTypes() throws IOException, NoSuchElementException {
         List<String> typeNames = Arrays.asList(types);
         assertTrue(typeNames.contains(testType.FEATURETYPENAME));
 
-        for (String typeName : types) {
```

Comment:
Hmm... I get this one had no assertions, but it was at least checking the code would not throw exceptions.

Reply:
I had to remove it, because it spent all too much time. I do think that it checks something that is checked elsewhere, but I will override in my class instead.

Reply:
Thanks!